packages/table-core/src/utils/getSortedRowModel.ts

```diff
@@ -45,7 +45,7 @@ export function getSortedRowModel<TData extends RowData>(): (
         const sortData = (rows: Row<TData>[]) => {
           // This will also perform a stable sorting using the row index
           // if needed.
-          const sortedData = rows.map(row => ({ ...row }))
+          const sortedData = rows.map(row => row.clone())
```

Comment:
If this is the only potential breaking change, I think we may still be able to ship it. It's on the edge of what's acceptable. We don't have much of a way to knowing how many people would rely on this.